accounts-db/src/tiered_storage/index.rs

```diff
@@ -38,6 +38,11 @@ pub enum AccountIndexFormat {
     AddressAndOffset = 0,
 }
 
+/// The position of an account/address entry in the accounts index block.
+/// This can be used to obtain the AccountOffset and address by looking through
+/// the accounts index block.
+pub struct IndexPosition(usize);
```

Comment:
We can still call it offset, but I am using `Position` here to improve clarity because its concept is slightly different from offset.  The number here is sequential: 0th, 1st, 2nd, 3rd, etc, which is different from an offset to a byte buffer.

What do you think?

Reply:
Changed to IndexOffset.